Gemfile

```diff
@@ -16,7 +16,6 @@ gem 'rack-cors', require: 'rack/cors'
 group :development, :test do
   gem 'simplecov', require: false
   gem 'pry-rails'
-  gem 'pry-byebug'
```

Comment:
There is no mention of removal of this gem in the PR description.

Reply:
Done, added comment in PR, thank you Dorian!